plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2954,25 +2955,26 @@ void producer_plugin_impl::produce_block() {
    chain.commit_block(br);
 
    const signed_block_ptr new_b = chain.head().block();
+   fc::time_point now = fc::time_point::now();
+
+   _time_tracker.add_other_time();
+   _time_tracker.report(new_b->block_num(), new_b->producer, now);
```

Comment:
We want to include the time for updating prometheus in `other`.

Reply:
https://github.com/AntelopeIO/spring/pull/881/commits/a0c811772ad6b1896840c83921f51b9c5dfc8ba8